### ISSUE TITLE
Additions to Command Line lesson

### ISSUE THREAD

**Issue by judytuna** [Techtonica/curriculum#534]
Here is information I found myself giving to apprentices / repeating while learning command line.  Please add the following to https://github.com/Techtonica/curriculum/blob/master/command-line/command-line-interface.md: 

- use tab for autocomplete. hit tab twice to see the list of autocomplete possibilities. if there are multiple matches, it'll autocomplete the string as far as it can before you have to disambiguate. encourage apprentices to stop typing and use tab.
- explain what we mean when we say "root" (later, disambiguate between "root" of your hard drive and "github project root") and that saying "root" is `\`
- `.` is current directory. i think the curriculum covers `..` as "one directory up" but let's have a section where we talk about `\` , `~`, `.`, and `..`
- type `open .` to see your current directory in finder. explain that `open` is not magically connected to Mac's Finder, but actually it's just using whatever default program on your Mac opens a file with that extension, and in the case of directories, the default program is Finder.
- absolute vs relative paths. you can always recognize an absolute path by the `\` (or `~`, which really itself has a `\` in it) in the front. else it's a relative path. emphasize that relative path is relative to your current directory. 
- you can pass in arguments to commands such as `ls` . clarify that typing `ls` alone is not magical, and that `ls` has a _default argument_ of `.` have them use `ls` to peek into other folders. 
- the mac keyboard shortcut `cmd + shift + g` can be used anywhere you see a Finder window (even in other Mac programs such as sourcetree or vs code when you're opening a file or directory). this brings up a little text box into which you can type or paste a path. use `pwd` to get your current path, then copy it from the Terminal and open up sourcetree, go to file > open, then use `cmd + shift + g` to paste in the path you want.

**Comment by srishtiBajpai** [comment]
hey @alodahl, i would like to do handle this issue

**Comment by saberprashant** [comment]
Hey @srishtiBajpai , have you completed the issue #578 ? Because we can only assign you another issue if you'll complete that one.
And, the evaluation will start from 1st March, so, do merge or release a PR of your code on or after 1st March, so that we can count that in your favor.
Thanks! :)

**Comment by srishtiBajpai** [comment]
hey @saberprashant I have completed the issue #578 and also I sent a PR for the same but I am afraid its too early for 1st march, will it still be counted?

**Comment by saberprashant** [comment]
Hi @srishtiBajpai , well yes it is too early to send a PR :) , but I am very happy to see your motivation. Keep it up like that. And, we will try our best to figure out what we can do in this situation. I think @alodahl can help you with that.

**Comment by saberprashant** [comment]
Hi @srishtiBajpai , we assigned you this issue because you are doing really great. But, please send a PR for this on 1st March, as we need to evaluate you according to the rules So, please send a PR accordingly.  
Thanks :)

**Comment by alodahl** [comment]
@srishtiBajpai , we cannot merge your PR for this until you address the review changes.
